hail/src/test/scala/is/hail/expr/ir/TakeByAggregatorSuite.scala

```diff
@@ -28,8 +29,8 @@ class TakeByAggregatorSuite extends HailSuite {
           Code.whileLoop(i < n.toLong,
             argR.invoke[Unit]("clear"),
             bytes := const("str").concat(i.toS).invoke[Array[Byte]]("getBytes"),
-            off := PBinary.allocate(argR, bytes.length()),
-            PBinary.store(off, bytes),
+            off := stringPT.allocate(argR, bytes.length()),
```

Comment:
I believe this is the correct type, PBinary seems wrong. Note that allocate and store instance methods on PString call the PBinary equivalent through fundamental type.

Reply:
the `store` methods on PString should take a Code[String], not a Code[Array[Byte]], I think